lib/sie/parser.rb

```diff
@@ -33,8 +37,14 @@ def parse(io)
 
     private
 
+    attr_reader :options
```

Comment:
Just to make sure I understand before I update: You prefer `attr_private :options`next to initialize rather than `attr_reader :options` among the other private methods?